examples/se_2_3_localization.cpp

```diff
@@ -86,15 +136,15 @@
  *  All these variables are summarized again as follows
  *
  *    X   : robot's extended pose, SE_2(3)
- *    u   : robot control input, (R_{k}.T v dt + 0.5*dt*dt*acc_{k}, omega_{k} dt, acc_{k} dt)
- *    Q   : control perturbation covariance
+ *    u   : robot control input, (Jinv * (R.T v dt + 0.5 dt^2 (alpha + R.T g)), omega dt, Jinv * ((alpha + R.T g) dt))
```

Comment:
I would not repeat here the expression of `u`. Instead, just add ", se_2(3)", akin to the line above

Reply:
better, use `u = u(X, y) \in se_2_3` with `X` state and `y=[a,w]` IMU readings, and point to expression (number) above in the doc.

Reply:
Fixed in https://github.com/artivis/manif/pull/154/commits/ded121b75f0f401c518336cf0d31ad862be50275